tests/helpers.py

```diff
@@ -230,3 +239,158 @@ def get_admin_key():
     out = ct_pod.exec_ceph_cmd('ceph auth get-key client.admin')
     base64_output = base64.b64encode(out['key'].encode()).decode()
     return base64_output
+
+
+def get_cephfs_data_pool_name():
+    """
+    Fetches ceph fs datapool name from Ceph
+
+    Returns:
+        str: fs datapool name
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    out = ct_pod.exec_ceph_cmd('ceph fs ls')
+    return out[0]['data_pools'][0]
+
+
+def validate_cephfilesystem(fs_name):
+    """
+     Verify CephFileSystem exists at ceph and k8s
+     Args:
```

Comment:
- Empty line between description and args
- Add return value + description

Reply:
Done fixed doc string